core_lib/data_layers/data/handler/sql_alchemy_data_handler_registry.py

```diff
@@ -54,7 +54,7 @@ def _on_db_session_exit(self, db_session: SqlAlchemyDataHandler):
         else:
             db_session.close()
 
-    def _create_engine(self, config):
+    def _create_engine(self, config) -> create_engine:

```

Comment:
this as rell return `engine`

another change please. 
just return the `create_engine` statment. variable `engine` not in use